contrib/stack/topsStack/Stack.py

```diff
@@ -358,7 +361,7 @@ def unpackSecondarysSLC(self,  stackReferenceDate, secondaryList, safe_dict):
             configObj.finalize()
 
             line_cnt += 1
-            line_cnt = configObj.write_wrapper_config2run_file(configName, line_cnt, self.numProcess)
+            line_cnt = configObj.write_wrapper_config2run_file(configName, line_cnt, secondary == secondaryList[-1], self.numProcess)
```

Comment:
Just tiny details: I would recommend using `enumerate` such as below to eliminate the `line_cnt=0` and `line_cnt+=1` lines. 
```
for i, secondary in enumerate(secondaryList):
    configObj.write_wrapper_config2run_file(configName, line_cnt=i, numProcess=self.numProcess)
```

Reply:
I got ahead of myself and missed some of your comments.

If we change the wrapper to default line_cnt = 1  

```
def write_wrapper_config2run_file(self, configName, line_cnt = 1, last_line = False, numProcess = 1):
```
the below works. I had to `line_cnt = i + 1` and `last_line = secondary == secondaryList[-1]` as `line_cnt` is now a keyword argument.

```
        # line_cnt = 0
        for i, secondary in enumerate(secondaryList):
            configName = os.path.join(self.config_path,'config_baseline_'+secondary)
            configObj = config(configName)
            configObj.configure(self)
            configObj.reference = os.path.join(self.work_dir,'reference/')
            configObj.secondary = os.path.join(self.work_dir,'secondarys/'+secondary)
            configObj.baselineFile = os.path.join(self.work_dir,'baselines/' + stackReferenceDate +'_' + secondary + '/' + stackReferenceDate +'_'+ secondary  + '.txt')
            configObj.computeAverageBaseline('[Function-1]')
            configObj.finalize()

            # line_cnt += 1
            configObj.write_wrapper_config2run_file(configName, line_cnt = i + 1, last_line = secondary == secondaryList[-1], numProcess = self.numProcess)
            del configObj
```

I am happy to remove the comment lines and do this in other similar locations if you like.

Reply:
That sounds good @RussellGrew. I would suggest using the below for clarity. 

```python
last_line = secondary == secondaryList[-1]
configObj.write_wrapper_config2run_file(configName, line_cnt=i+1, last_line=last_line, numProcess=self.numProcess)

```